### ISSUE TITLE
Activity box shows unexpected content

### ISSUE THREAD

**Issue by fredewe** [liqd/adhocracy-plus#126]
**URL:** https://adhocracy.plus/test/
**user:** -
**expected behaviour:** as a user i expect to see only informations about the participation.
**behaviour:** also technical information about the modules are shown (a4_candy_polls:voting) 
**important screensize:**
**device & browser:** 
![Bildschirmfoto 2019-12-02 um 11 33 31](https://user-images.githubusercontent.com/5470479/69952877-66f50500-14f8-11ea-92db-8326a62020d5.png)

**Comment/Question:** 

Screenshot?

**Comment by phillimorland** [comment]
We might not keep the activity box so I have put this on hold for now but when looking into it, it seems activity box and local time seem to also be broken and you can start participating in a project before it has officially started, have made another issue that I linked below.

**Comment by fuzzylogic2000** [comment]
We need to check this in the user profile. The activity box is gone, though!

**Comment by fuzzylogic2000** [comment]
I will check!

**Comment by fredewe** [comment]
@fuzzylogic2000 Don't you think we can close this one?

**Comment by fuzzylogic2000** [comment]
@fredewe Yes, I just wanted to check if we still have this issue in the user profile and close if not! So keep it until tomorrow. :)
